### ISSUE TITLE
ArchiveServiceTest : Remove unused imports

### ISSUE THREAD

**Issue by rohanKanojia** [fabric8io/docker-maven-plugin#1743]
### Description

ArchiveServiceTest seems to be having some imports that are not used anymore. Probably these were not updated while we refactored tests to use JUnit5.

https://github.com/fabric8io/docker-maven-plugin/blob/267e4674e63b8800531bb1ce4a4e2b0b31e8007d/src/test/java/io/fabric8/maven/docker/service/ArchiveServiceTest.java#L3

https://github.com/fabric8io/docker-maven-plugin/blob/267e4674e63b8800531bb1ce4a4e2b0b31e8007d/src/test/java/io/fabric8/maven/docker/service/ArchiveServiceTest.java#L8-L9

### Before You Start
Make sure you read [CONTRIBUTING.md ](https://github.com/fabric8io/docker-maven-plugin/blob/master/CONTRIBUTING.md).

### Acceptance Criteria
- [ ] All unused imports are removed from ArchiveServiceTest

**Comment by rohanKanojia** [comment]
@mdxabu : This might help you in getting started.

Let me know if it is clear enough for you.

**Comment by mdxabu** [comment]
No, you can give some more little explanation and i will start my contribution

**Comment by rohanKanojia** [comment]
@mdxabu The lines I pointed out in the issue description don't seem to be getting used in the test class. 

You need to remove those lines. If you're using any IDE (Eclipse or IntelliJ) they would automatically highlight this issue.

**Comment by rohanKanojia** [comment]
@mdxabu : If you are new to contributing to a Java project, I've written a related [blogpost](https://rohaan.medium.com/how-to-contribute-to-eclipse-jkube-f3bc7f902bf4) about this. It's about a different project but should give you some idea on how to set up project on your system.

**Comment by mdxabu** [comment]
Ok, I will work on this task.

**Comment by mdxabu** [comment]
@rohanKanojia , I don't know why I got these errors when I use ```mvn clean install```
```
[ERROR]   RunServiceTest.shutdownWithKeepingContainer ┬╗ Mockito 
Mockito cannot mock thi...
[ERROR]   RunServiceTest.shutdownWithPreStopExecConfig ┬╗ Mockito 
Mockito cannot mock th...
[ERROR]   RunServiceTest.shutdownWithoutKeepingContainers ┬╗ Mockito 
Mockito cannot mock...
[ERROR]   RunServiceTest.shutdownWithoutKeepingContainersAndRemovingVolumes ┬╗ Mockito 
M...
[ERROR]   RunServiceTest.testCreateContainerAllConfig ┬╗ Mockito 
Mockito cannot mock thi...
[ERROR]   RunServiceTest.testStopModeWithKill ┬╗ Mockito 
Mockito cannot mock this class:...
[ERROR]   RunServiceTest.testVolumesDuringStart ┬╗ Mockito 
Mockito cannot mock this clas...
[ERROR]   RunServiceTest.testWithException ┬╗ Mockito 
Mockito cannot mock this class: cl...
[ERROR]   RunServiceTest.testWithMultipleStopExceptions ┬╗ Mockito 
Mockito cannot mock t...
[ERROR]   RunServiceTest.testWithoutWait ┬╗ Mockito 
Mockito cannot mock this class: clas...
[ERROR]   WatchServiceTest.testWatchCallsGetAllAssemblyConfigurations ┬╗ Mockito 
Mockito...
[ERROR]   StartContainerExecutorTest.testStartContainers ┬╗ ParameterResolution Failed to...
[ERROR]   AuthConfigFactoryTest.awsStaticCredentialsArePickedUpFromEnvironment ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.awsTemporaryCredentialsArePickedUpFromEnvironment ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.ecsTaskRole ┬╗ NoClassDefFound Could not initialize class...
[ERROR]   AuthConfigFactoryTest.fargateTaskRole ┬╗ NoClassDefFound Could not initialize c...
[ERROR]   AuthConfigFactoryTest.getAuthConfigViaAwsSdk ┬╗ NoClassDefFound Could not initi...
[ERROR]   AuthConfigFactoryTest.hasAuthForRegistryInDockerConfig_whenDefaultDockerRegistryInDockerConfig_thenReturnTrue ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.hasAuthForRegistryInDockerConfig_whenRegistryAbsentInDockerConfig_thenReturnFalse ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.hasAuthForRegistryInDockerConfig_whenRegistryInDockerConfigButDifferentUser_thenReturnFalse ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.hasAuthForRegistryInDockerConfig_whenRegistryInDockerConfig_thenReturnTrue ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.incompleteAwsCredentialsAreIgnored ┬╗ NoClassDefFound Cou...
[ERROR]   AuthConfigFactoryTest.testDockerAuthLogin ┬╗ NoClassDefFound Could not initiali...
[ERROR]   AuthConfigFactoryTest.testDockerLoginDefaultToCredentialsStore ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.testDockerLoginFallsBackToAuthWhenCredentialHelperDoesNotMatchDomain ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.testDockerLoginNoAuthConfigFoundWhenCredentialHelperDoesNotMatchDomainOrAuth ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.testDockerLoginNoConfig ┬╗ NoClassDefFound Could not init...
[ERROR]   AuthConfigFactoryTest.testDockerLoginNoErrorWhenEmailInAuthConfigContainsNullValue ┬╗ NoClassDefFound
[ERROR]   AuthConfigFactoryTest.testDockerLoginSelectCredentialHelper ┬╗ NoClassDefFound ...
[ERROR]   AuthConfigFactoryTest.testDockerLoginSelectCredentialsStore ┬╗ NoClassDefFound ...
[ERROR]   AuthConfigFactoryTest.testDockerSystemProperty ┬╗ NoClassDefFound Could not ini...
[ERROR]   AuthConfigFactoryTest.testDockerSystemPropertyHasPrecedence ┬╗ NoClassDefFound ...
[ERROR]   AuthConfigFactoryTest.testEmpty ┬╗ NoClassDefFound Could not initialize class u...
[ERROR]   AuthConfigFactoryTest.testFromPluginConfiguration ┬╗ NoClassDefFound Could not ...
[ERROR]   AuthConfigFactoryTest.testFromPluginConfigurationFailed ┬╗ NoClassDefFound Coul...
[ERROR]   AuthConfigFactoryTest.testFromPluginConfigurationPull ┬╗ NoClassDefFound Could ...
[ERROR]   AuthConfigFactoryTest.testFromSettingsDefault ┬╗ NoClassDefFound Could not init...
[ERROR]   AuthConfigFactoryTest.testFromSettingsDefault2 ┬╗ NoClassDefFound Could not ini...
[ERROR]   AuthConfigFactoryTest.testFromSettingsSimple ┬╗ NoClassDefFound Could not initi...
[ERROR]   AuthConfigFactoryTest.testOpenShiftConfigFromPluginConfig ┬╗ NoClassDefFound Co...
[ERROR]   AuthConfigFactoryTest.testOpenShiftConfigFromSystemProps ┬╗ NoClassDefFound Cou...
[ERROR]   AuthConfigFactoryTest.testOpenShiftConfigFromSystemPropsNegative ┬╗ ExceptionInInitializer
[ERROR]   AuthConfigFactoryTest.testOpenShiftConfigNotLoggedIn ┬╗ NoClassDefFound Could n...
[ERROR]   AuthConfigFactoryTest.testRegistrySystemProperty ┬╗ NoClassDefFound Could not i...
[ERROR]   AuthConfigFactoryTest.testSystemProperty ┬╗ NoClassDefFound Could not initializ...
[ERROR]   AuthConfigFactoryTest.testSystemPropertyNoPassword ┬╗ NoClassDefFound Could not...
[ERROR]   AuthConfigFactoryTest.testWrongUserName ┬╗ NoClassDefFound Could not initialize...
[ERROR]   ImageNameFormatterTest.artifact ┬╗ Mockito 
Mockito cannot mock this class: cla...
[ERROR]   ImageNameFormatterTest.defaultUserName ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.defaultUserName ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.defaultUserName ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.defaultUserName ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.defaultUserName ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.defaultUserName ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.groupIdWithProperty ┬╗ Mockito 
Mockito cannot mock this...
[ERROR]   ImageNameFormatterTest.invalidFormatChar ┬╗ Mockito 
Mockito cannot mock this c...
[ERROR]   ImageNameFormatterTest.nonSnapshotArtifact ┬╗ Mockito 
Mockito cannot mock this...
[ERROR]   ImageNameFormatterTest.simple ┬╗ Mockito 
Mockito cannot mock this class: class...
[ERROR]   ImageNameFormatterTest.tag ┬╗ Mockito 
Mockito cannot mock this class: class or...
[ERROR]   ImageNameFormatterTest.tagWithDockerImageTagSet ┬╗ Mockito 
Mockito cannot mock...
[ERROR]   ImageNameFormatterTest.tagWithProperty ┬╗ Mockito 
Mockito cannot mock this cla...
[ERROR]   ImageNameFormatterTest.timestamp ┬╗ Mockito 
Mockito cannot mock this class: cl...
[ERROR]   StartOrderResolverTest.circularDep ┬╗ Mockito 
Mockito cannot mock this class: ...
[ERROR]   StartOrderResolverTest.simple ┬╗ Mockito 
Mockito cannot mock this class: class...
[ERROR]   AwsSdkAuthConfigFactoryTest.nullValueIsPassedOn ┬╗ NoClassDefFound Could not in...
[ERROR]   AwsSdkAuthConfigFactoryTest.reflectionWorksForBasicCredentials ┬╗ NoClassDefFound
[ERROR]   AwsSdkAuthConfigFactoryTest.reflectionWorksForSessionCredentials ┬╗ NoClassDefFound
[ERROR]   ExitCodeCheckerTest.checkReturnsFalseIfActualExitCodeDoesNotMatchExpectedExitCode ┬╗ Mockito
[ERROR]   ExitCodeCheckerTest.checkReturnsFalseIfContainerDoesNotExist ┬╗ Mockito 
Mockit...
[ERROR]   ExitCodeCheckerTest.checkReturnsFalseIfContainerIsStillRunning ┬╗ Mockito 
Mock...
[ERROR]   ExitCodeCheckerTest.checkReturnsTrueIfActualExitCodeMatchesExpectedExitCode ┬╗ Mockito
[ERROR]   HttpPingCheckerTest.checkingAfterIOExceptionFails:81 Mockito 
Mockito cannot m...
[ERROR]   HttpPingCheckerTest.checkingAfterSuccessfulResponseSucceeds:43 Mockito 
Mockit...
[ERROR]   HttpPingCheckerTest.checkingAfterUnsuccessfulResponseFails:62 Mockito 
Mockito...
[INFO]
[ERROR] Tests run: 871, Failures: 0, Errors: 372, Skipped: 6
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:26 min
[INFO] Finished at: 2024-02-03T14:40:31+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M2:test (default-test) on project docker-maven-plugin: There are test failures.
[ERROR]
[ERROR] Please refer to D:\Abu\Github-Projects\docker-maven-plugin\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by rohanKanojia** [comment]
@mdxabu : Which JDK are you using? We are currently on JDK8

**Comment by mdxabu** [comment]
JDK 21

**Comment by rohanKanojia** [comment]
@mdxabu : Could you please install JDK8 or JDK11 and see if problem persists?

**Comment by mdxabu** [comment]
I changed to jdk8 but problem does not solved

**Comment by rohanKanojia** [comment]
@mdxabu : Why do you want to add a dependency?

You need to do the following things:
1. `mvn clean install -DskipTests` 
2. Open IntelliJ and Load as Maven Project

**Comment by mdxabu** [comment]
> @mdxabu : Why do you want to add a dependency?
> 
> You need to do the following things:
> 
> 1. `mvn clean install -DskipTests`
> 2. Open IntelliJ and Load as Maven Project

I try this, but it doesn't work.
Not for adding a dependency, just asking why Intellij idea does not find a plugin/dependency.

**Comment by mdxabu** [comment]
@rohanKanojia , can you help me to fix this.

**Comment by rohanKanojia** [comment]
Fixed by #1744
